bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -155,16 +155,25 @@ metadata:
     createdAt: "2020-09-08T12:21:00Z"
     description: OADP (OpenShift API for Data Protection) operator sets up and installs
       Data Protection Applications on the OpenShift platform.
-    features.operators.openshift.io/token-auth-aws: "true"
     olm.skipRange: '>=0.0.0 <99.0.0'
     operatorframework.io/suggested-namespace: openshift-adp
-    operators.openshift.io/infrastructure-features: '["Disconnected"]'
     operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
       Platform Plus"]'
     operators.operatorframework.io/builder: operator-sdk-v1.23.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/oadp-operator
     support: Red Hat
+    operators.openshift.io/infrastructure-features: '["Disconnected"]'
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
```

Comment:
Are we gonna flip the FIPS annotation to true later ?

Reply:
ok, got it ! just read earlier comments :)

Reply:
Apologies for the lack of additional detail.  I thought your previous follow-up comment about reading through the PR comments was sufficient. :sweat_smile: 

Per Wes's earlier comment:
> FIPS in 1.3.x and 1.2.x on hold for QE validation

We'll defer enabling ("true") this value on for release until QE has signed-off on OADP being FIPS compliant.